modules/edge-governance/src/governance.rs

```diff
@@ -210,10 +216,12 @@ decl_storage! {
 		/// A list of all extant proposals.
 		pub Proposals get(proposals): Vec<T::Hash>;
 		/// A list of active proposals along with the time at which they complete.
-		pub ActiveProposals get(active_proposals): Vec<(T::Hash, T::Moment)>;
+		pub ActiveProposals get(active_proposals): Vec<(T::Hash, T::BlockNumber)>;
 		/// Amount of time a proposal remains in "Voting" stage.
-		pub VotingTime get(voting_time) config(): T::Moment;
-		/// Map for retrieving the information about any proposal from its hash.
-		pub ProposalOf get(proposal_of): map T::Hash => Option<ProposalRecord<T::AccountId, T::Moment>>;
+		pub VotingTime get(voting_time) config(): T::BlockNumber;
+		/// Map for retrieving the information about any proposal from its hash. 
+		pub ProposalOf get(proposal_of): map T::Hash => Option<ProposalRecord<T::AccountId, T::BlockNumber>>;
+		/// Registration bond
+		pub ProposalCreationBond get(proposal_creation_bond) config(): BalanceOf<T>;
```

Comment:
Might make sense to have a default value for this, like here: https://github.com/paritytech/substrate/blob/master/srml/council/src/seats.rs#L316

Reply:
Good idea